submissions/yusiratArogundade/HTML/index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>My Product Page</title>
+</head>
+<body>
+	<div>
+		<header>
+			<section>			
+				<h1>INDULGENCE FASHION HUB</h1>			
+				<input type="search"> <button>search</button>
+			</section>			
+		</header>
+	</div>
+	<div>
+		<section>
+			<nav>
+			<table>
+				<tr><th> <a href=""><h3>About us</h3></a></th>
+					<th><a href=""><h3>Men Wears</h3></a></th>
+					<th><a href=""><h3>Women Wears</h3></a></th>
+					<th> <a href=""><h3>Baby Wears</h3></a></th>
+				</tr>
+			</table>			
+		</nav>
+		</section>		
+	</div>
+	<hr>
+	<div>
+		<section>
+			<h2>Baby Wears</h2>
+			<table>
+				<tr>
+					<th><img src="baby.png"></th>
+					<th><table>
+						<tr>
+							<th><h2>Baby Girl Ankara One Hand A-Cut Dress</h2></th>
+						</tr>
+						<tr>
+							<th><h3>Descriptiion</h3>
+								<ul>
+									<li>Beautiful African prints and soft cotton combo</li>
+									<li>Smart and simple</li>
+									<li>Classy hand crafted stylish designs for infants and  kids.</li>
+									<li>Very affordable, breathable and comfortable to wear in hot/dry seasons</li>
+									<li>It can be adjusted to fit as desired as it can suit a wide range of body length and sizes. </li>
+									<li>The materials are genuine DaViva and Woodin prints.</li>
+									<li>Durable, long lasting, and machine washable.</li>
+								</ul>
+							</th>
+							<tr><th><h4><strong>Price &#8358; 4,500</strong></h4></th></tr>
+						</tr>
+					</table>						
+					</th>
+				</tr>
+			</table>
+		</section>
+	</div>
+	<div>
+		<section>			
+			<table>
+				<tr>
+					<th><img src="baby2.png"></th>
+					<th><table>
+						<tr>
+							<th><h2>Flower Girls Pleated Dress </h2></th>
+						</tr>
+						<tr>
+							<th><h3>Descriptiion</h3>
+								<ul>
+									<li>Soft Material and first class workmanship.</li>
+									<li>Loose and comfortable</li>
+									<li>Daily Wear, School, Party Costume, Best gift for Girl</li>
+									<li>Material: Polyester + Cotton Blend</li>
+									<li>Sleeve: Sleeveless</li>
+								</ul>
+							</th>
+							<tr><th><h4><strong>Price &#8358; 5,500</strong></h4></th></tr>
+						</tr>
+					</table>						
+					</th>
+				</tr>
+			</table>
+		</section>
+	</div>
+	<div>
+		<h3>Similar Product</h3>
+		<ul>
```

Comment:
@yusrah247 You are meant to use ordered list to render the similar products not unordered list.